config/templates/lambda.tpl.js

```diff
@@ -1,4 +1,4 @@
-export function handler(event, context) {
+export function handler(event, context, callback) {
     console.log(event);
-    context.success('done!');
-}
\ No newline at end of file
+    callback(null, 'done!');
```

Comment:
Yep amazon changed their API when they updated to node4.3.

[Transitioning Lambda Function Code to Node.js Runtime v4.3](http://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-using-old-runtime.html#transition-to-new-nodejs-runtime)